src/cljs/org/broadinstitute/firecloud_ui/page/workspace/method_config_importer.cljs

```diff
@@ -35,119 +35,120 @@
        :owner (rand-nth ["thibault@broadinstitute.org" "esalinas@broadinstitute.org"  ])})
     (range (rand-int 50))))
 
+; Convenience method for rendering a label and text
```

Comment:
(my philosophy follows, which is not set in stone)
The code should explain what. The comments should explain why. In this case, your code is clear, so you don't need this comment. If you did need the comment, it would be better to rewrite the code so it is more clear.